ui/apps/dashboard/src/utils/urls.ts

```diff
@@ -14,3 +16,38 @@ export function getManageKey(pathname: string) {
     return null;
   }
 }
+
+export const pathCreator = {
+  deploys({ envSlug }: { envSlug: string }): Route {
+    // @ts-expect-error
+    return `/env/${envSlug}/deploys`;
+  },
+  deploy({ deployID, envSlug }: { deployID: string; envSlug: string }): Route {
+    // @ts-expect-error
+    return `/env/${envSlug}/deploys/${deployID}`;
+  },
+  events({ envSlug }: { envSlug: string }): Route {
+    // @ts-expect-error
+    return `/env/${envSlug}/events`;
+  },
+  eventType({ envSlug }: { envSlug: string; eventName: string }): Route {
```

Comment:
`eventName` is missing, even though it's later used.

```suggestion
  eventType({ envSlug, eventName }: { envSlug: string; eventName: string }): Route {
```